### ISSUE TITLE
Timer is implemented incorrectly

### ISSUE THREAD

**Issue by FRC-4509** [FRC-4509-MechBulls/PizzaBot-2017#4]
The way that the Timer class is implemented in methods such as `driveForFeet` is incorrect, causing those methods to run infinitely.

**Comment by agoeden12** [comment]
You are correct. Although no proper testing was done with the `driveForFeet()` so theoretically if we fixed the issue with the Timer class implementation then `driveForFeet()` would work.

However, we will leave that function alone for now because we are trying to get encoders, calculates the distance for us, to get an accurate measurement every time for us to drive a certain distance without the complex code in `driveForFeet()`.
